content/issue-1/contribute-to-rustc.md

```diff
@@ -0,0 +1,105 @@
+# I Learn Rust By Contributing To Rust Compiler
+
+---
```

Comment:
We can delete those lines. BTW, I suggest renaming the title to "How I contribute to Rust Compiler". Any thoughts? @LCrossman @macko76

Reply:
yes, I didn't come with a good subject, any better one?